### PR TITLE
Use correct names for both versions

### PR DESCRIPTION
**Pull request by gdi2290**
the latest version of angular prefers to just be called `angular`, as the version increases every 6 months, while the original one remains with `JS` appended

**[Insert URL to the list here.]**
https://github.com/AngularClass/awesome-angular

**[Explain what this list is about and why it should be included here.]**
repo renamed

# By submitting this pull request I confirm I've read and complied with the below requirements.

Failure to properly do so will just result in the pull request being closed and everyone's time wasted. Please read it twice. Most people miss many things.

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title. For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should include a short description about the list project/theme (not the list itself), and should be added at the bottom of the appropriate category.
- The list I added
	- **has been around for at least 30 days,**
	- is the result of hard work and is the best I could possibly produce,
	- is a non-generated Markdown file in a GitHub repo,
	- is not a duplicate,
	- includes a succinct description of the project/theme at the top ([example](https://github.com/willempienaar/awesome-quantified-self)),
	- includes a project logo/illustration (whenever possible) at the top right of the readme ([example](https://github.com/sindresorhus/awesome-electron)). (The logo should link to the project website or any relevant website. The logo should be high-DPI, so set it to maximum half the width of the original image.),
	- only has awesome items **with descriptions** *(Awesome lists are curations of the best, not of everything)*,
	- contains the [awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- has a Table of Contents section **named `Contents`** as the first section,
	- has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license) *(Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc)* *(If you use a license badge, it should be SVG, not PNG)*,
	- has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines),
	- doesn't include a Travis badge,
	- has been checked for inconsistent formatting, spelling, and grammar:
		- Each link description starts with an uppercase character and ends with a period.
		- Drop all the `A` / `An` prefixes in the descriptions.
		- Consistent naming. For example, `Node.js`, not `NodeJS` or `node.js`.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -161,8 +161,8 @@
 	- [Relay](https://github.com/expede/awesome-relay) - Framework for building data-driven React apps.
 - [Web Components](https://github.com/mateusortiz/webcomponents-the-right-way)
 - [Polymer](https://github.com/Granze/awesome-polymer)
-- [Angular 2](https://github.com/AngularClass/awesome-angular2) - App framework.
-- [Angular](https://github.com/gianarb/awesome-angularjs) - App framework.
+- [Angular](https://github.com/AngularClass/awesome-angular) - App framework.
```
Comment (gdi2290) [inline readme.md]: I renamed the repo to drop the 2

**Comment by davisonio** [comment]
Nice fix this aligns with the naming from Angular's official blog:

> As you might have already guessed, the term “Angular 2” is also kind of deprecated once we get to version 4, 5 etc. That said, we should start naming it simply “Angular” without the version suffix.

http://angularjs.blogspot.co.uk/2016/12/ok-let-me-explain-its-going-to-be.html

Merged in ✨

**Comment by gdi2290** [comment]
🐈 thanks

**Comment by sindresorhus** [comment]
I think this makes it really unclear. Most people won't know the difference...

**Comment by sindresorhus** [comment]
The PR title is also not very useful.

**Comment by AnthonyTsang** [comment]
@sindresorhus I dun think it is something we can control. The name is officially announced and used in all of the places like documents and tutorials. It causes by the problem that AngularJS is so much different to Angular that it is not backward compatible at all. But it is a decision made by the angular team and they need to accept the consequence.

**Comment by sindresorhus** [comment]
@AnthonyTsang I'm not talking about the name, but rather having `Angular` and `AngularJS` is not very user-friendly, as it's hard to know the difference. But doesn't matter, I removed the Angular v1 list https://github.com/sindresorhus/awesome/commit/f83c5da006537ea9451ad4ee1f81e1049b54d872.

**Comment by gdi2290** [comment]
I'll mention the AngularJS version in the Angular version then

**Pull request merged**